src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -124,6 +124,24 @@ public void chooseMcqOption(int qnNumber, int responseNumber, String choiceName)
         click(element);
     }
 
+    public boolean doesMcqOptionExist(int qnNumber, int responseNumber, String choiceName) {
+        try {
+            chooseMcqOption(qnNumber, responseNumber, choiceName);
```

Comment:
Verifying a choice exists by choosing it is a kind of hack. Is there a cleaner method?

Reply:
How's the method below? 
The clicking action is no longer present. 
```
public boolean checkIfMcqOrMsqChoiceExists(int qnNumber, int responseNumber, String choiceName) {
    String name = Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber;
    name = SanitizationHelper.sanitizeStringForXPath(name);
    String sanitizedChoiceName = SanitizationHelper.sanitizeStringForXPath(choiceName);
    try{
        browser.driver.findElement(
                By.xpath("//input[@name=" + name + " and @value=" + sanitizedChoiceName + "]"));
    } catch(NoSuchElementException e) {
        return false;
    }
    return true;
}
```